SVN revision 758314
Author: dfaure <dfaure@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date:   Mon Jan 7 15:54:16 2008 +0000

    doesn't need to link to libkonq anymore

SVN revision 758304
Author: dfaure <dfaure@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date:   Mon Jan 7 15:14:42 2008 +0000

    doesn't use libkonq anymore

diff --git a/kdebase/apps/konqueror/shellcmdplugin/CMakeLists.txt b/kdebase/apps/konqueror/shellcmdplugin/CMakeLists.txt
index ddbad19..64919ce 100644
--- a/kdebase/apps/konqueror/shellcmdplugin/CMakeLists.txt
+++ b/kdebase/apps/konqueror/shellcmdplugin/CMakeLists.txt
@@ -7,7 +7,7 @@ set(konq_shellcmdplugin_PART_SRCS kshellcmdexecutor.cpp kshellcmddialog.cpp kshe
 kde4_add_plugin(konq_shellcmdplugin ${konq_shellcmdplugin_PART_SRCS})
 
 
-target_link_libraries(konq_shellcmdplugin  konq ${KDE4_KDEUI_LIBS} ${KDE4_KDESU_LIBS})
+target_link_libraries(konq_shellcmdplugin ${KDE4_KPARTS_LIBS} ${KDE4_KDESU_LIBS})
 
 install(TARGETS konq_shellcmdplugin  DESTINATION ${PLUGIN_INSTALL_DIR} )
 
diff --git a/kdebase/apps/konqueror/remoteencodingplugin/CMakeLists.txt b/kdebase/apps/konqueror/remoteencodingplugin/CMakeLists.txt
index bb92516..160f1a2 100644
--- a/kdebase/apps/konqueror/remoteencodingplugin/CMakeLists.txt
+++ b/kdebase/apps/konqueror/remoteencodingplugin/CMakeLists.txt
@@ -6,7 +6,7 @@ set(konq_remoteencoding_PART_SRCS kremoteencodingplugin.cpp )
 kde4_add_plugin(konq_remoteencoding ${konq_remoteencoding_PART_SRCS})
 
 
-target_link_libraries(konq_remoteencoding  ${KDE4_KDECORE_LIBS} konq )
+target_link_libraries(konq_remoteencoding  ${KDE4_KPARTS_LIBS} )
 
 install(TARGETS konq_remoteencoding  DESTINATION ${PLUGIN_INSTALL_DIR} )
 
